distribution/kernel/pom.xml

```diff
@@ -165,6 +170,12 @@
                                     <destFileName>ddf-branding.jar</destFileName>
                                 </artifactItem>
                                 <artifactItem>
+                                    <groupId>org.ops4j.pax.logging</groupId>
+                                    <artifactId>pax-logging-log4j2</artifactId>
+                                    <version>${pax.logging.version}</version>
+                                    <outputDirectory>${setup.folder}/system/org/ops4j/pax/logging/pax-logging-log4j2/${pax.logging.version}</outputDirectory>
```

Comment:
This seems very weird to copy into the system folder this way.  Can we add the new version of pax logging to our platform app so it is including the same way as everything else in the system folder?  We should add some comments with a JIRA number for when we upgrade Karaf with the correct version of pax logging so we can easily remove all of this.